open-craft/blockstore
# Configuration

## Description

- Adds a docker container for doing dev which runs as part of the `devstack_default` network and is accessible with the hostname `edx.devstack.blockstore` from devstack containers.
- Add CircleCI config with tests and quality jobs.
- Adds `diff-cover`. On CircleCI the coverage details can be seen in tests logs.
- Updates commands in Makefile.
- Adds description and dev setup details in README.
- Changes `pep8` to `pycodestyle`.
- Fixes all quality issues.

## Testing Instructions
- Verify that the tests and quality jobs run on CircleCI.
- Verify the the docker container is setup correctly and can be accessed from devstack containers.
- Verify diff-cover is logged correctly.
- Verify the make commands all run correctly.

The last commit has been added to test the quality job. It will be removed before merging.
